block-storage-attacher/tests/e2e/conf/prod-us-east.conf

```diff
@@ -41,18 +41,18 @@ tor01_PVG_CRUISER_PRIVATE_VLAN=2259903
 
 # For plugin build
 GOPATH=$GOPATH  					# Read from environment
-BLOCK_PLUGIN_HOME=$GOPATH/src/github.ibm.com/alchemy-containers/block-storage-attacher/tests/e2e/
+BLOCK_PLUGIN_HOME=$GOPATH/src/github.ibm.com/alchemy-containers/ibmcloud-storage-utilities/block-storage-attacher/tests/e2e/
```

Comment:
This should be src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher.
Check the above new path and change accordingly.